gemfiles/.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_RETRY: "1"
```

Comment:
I think this should not be part of the project, should it be needed for tests, it could be added as a command line argument

Reply:
Good point! Removed it.